formwidgets/ActionBuilder.php

```diff
@@ -245,17 +246,21 @@ public function setCacheActionData($action)
 
         $cache[$action->id] = json_encode($this->makeCacheActionData($action));
 
-        $_POST['action_data'] = $cache;
+        $requestData = Request::all();
+        array_set($requestData, 'action_data' , $cache);
+        Request::merge($requestData);
```

Comment:
@fansaien As per [this comment](https://github.com/rainlab/notify-plugin/pull/16#issuecomment-495843838), could you try rewriting the above as:

```php
Request::merge([
    'action_data' => $cache
]);
```

Reply:
Please review the new version, thanks